src/utils/command.rs

```diff
@@ -103,14 +114,24 @@ pub trait BotCommand: Sized {
 /// ```
 /// use teloxide::utils::command::parse_command;
 ///
-/// let text = "/mute 5 hours";
-/// let (command, args) = parse_command(text).unwrap();
+/// let text = "/mute@my_admin_bot 5 hours";
+/// let (command, args) = parse_command(text, "my_admin_bot").unwrap();
 /// assert_eq!(command, "/mute");
 /// assert_eq!(args, vec!["5", "hours"]);
 /// ```
-pub fn parse_command(text: &str) -> Option<(&str, Vec<&str>)> {
+pub fn parse_command<N>(text: &str, bot_name: N) -> Option<(&str, Vec<&str>)>
+where
+    N: Into<String>,
```

Comment:
Do we really need `String` in such places? I think this could be replaced with smth like `AsRef<str>`

Reply:
We always use `Into<String>` because it is simply simpler)

Reply:
`AsRef<str>` is not that complicated. And since we code Rust, we probably want as few stupid allocations as possible, especially when they can be fixed easilly

Reply:
Do you sure that `AsRef<str>` won't spread lifetimes across the library?

Reply:
I think in this exact place (and anywhere where we just compare `str`-s, some non-storing stuff) there will be no explicit lifetimes

Reply:
Nice catch!